client/containers/DashboardOverview/overviewRows/OverviewRowSkeleton.tsx

```diff
@@ -22,7 +16,7 @@ type Props = {
 	byline?: React.ReactNode;
 	rightElement?: React.ReactNode;
 	darkenRightIcons?: boolean;
-	iconLabelPairs: IconLabelPair[];
+	iconLabelPairs: React.ReactNode;
```

Comment:
Since this can now take _any_ content, `iconLabelPairs` is too specific a name. Maybe just `details`?